### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by huqiaohong** [datawhalechina/self-llm#378]
在进行Qwen2-VL的lora微调案例-LaTexOCR时，运行最后的train出现这个警告：0% 0/124 [00:00<?, ?it/s]/usr/local/lib/python3.11/dist-packages/torch/utils/checkpoint.py:87: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
进度条一直为0，请问该如何解决

**Comment by FrancesPlus** [comment]
你好，请问你解决了吗，我今天在尝试qwen3微调案例的时候，也出现了这个问题，导致无法推理出正确答案。
/usr/local/lib/python3.12/dist-packages/torch/utils/checkpoint.py:85: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
